### PR TITLE
How to start Grafana for the first time

### LINKED ISSUE #11580: Docs feedback: guides/getting_started.md
Read before posting:

### PR DESCRIPTION
**Pull request by Ijin08**
Added guide for logging in to Grafana for the first and how to add a datasource.

fixes  #11580

### CODE REVIEW

File: docs/sources/guides/getting_started.md [hunk 1]
```diff
@@ -15,6 +15,31 @@ weight = 1
 
 This guide will help you get started and acquainted with Grafana. It assumes you have a working Grafana server up and running and have added at least one [Data Source](/features/datasources/).
 
+## Logging in for the first time
+
+To run Grafana open your browser and go to port 3000 which is the default port. If you have changed the port you go to that port. There you will see the login page. User name is admin and password is admin. When you log in for the first time you will be asked to change your password. You can later go to user preferences and change your user name.
```
Comment (marefr) [inline docs/sources/guides/getting_started.md]: Would like to suggest some changes:

Default username is `admin` and default password is `admin`. When you log in for the first time you will be asked to change your password. We strongly encourage you to follow Grafana’s best practices and change the default administrator password. You can later go to user preferences and change your user name.

File: docs/sources/guides/getting_started.md [hunk 2]
```diff
@@ -15,6 +15,31 @@ weight = 1
...
+
+
+## How to add a data source
+
+{{< docs-imagebox img="/img/docs/v52/sidemenu-datasource.png" max-width="250px" class="docs-image--right docs-image--no-shadow">}}
+
+Before you create your first dashboard you need to add your data source.
+
+First move your cursor to the cog on the side menu which will show you the configuration menu. If the side menu is not visible click the Grafana icon in the upper left corner. The first item on the configuration menu is data sources. Click and you will come to data sources. You can also simply click the cog.
```
Comment (marefr) [inline docs/sources/guides/getting_started.md]: > The first item on the configuration menu is data sources. Click and you will come to data sources.

The first item on the configuration menu is data sources, click on that and you'll be taken to the data sources page where you can add and edit data sources.

File: docs/sources/guides/getting_started.md [hunk 3]
```diff
@@ -15,6 +15,31 @@ weight = 1
...
+
+
+Click Add data source and you will come to the settings page of your new data source.
+
+{{< docs-imagebox img="/img/docs/v52/add-datasource.png" max-width="700px" class="docs-image--no-shadow">}}
+
+The first thing you will do is give the data source a name and select the right type.
```
Comment (marefr) [inline docs/sources/guides/getting_started.md]: > The first thing you will do is give the data source a name and select the right type.

First, give the data source a `Name` and then select which `Type` of data source you'll want to create, see [Supported data sources](/features/datasources/#supported-data-sources) for more information.

File: docs/sources/guides/getting_started.md [hunk 4]
```diff
@@ -15,6 +15,31 @@ weight = 1
...
+Next you need to specify the data sources HTTP URL and how you will access the data source.
```
Comment (marefr) [inline docs/sources/guides/getting_started.md]: This is only true for some datasources. Maybe possible to combine link to Supported data sources above with details on how to configure each?

Not sure, but as a last resort if you don't have any data source to connect to yet maybe link to the Grafana testdata datasource features/datasources/testdata/?

File: docs/sources/installation/windows.md [hunk 5]
```diff
@@ -43,3 +43,9 @@ Read more about the [configuration options]({{< relref "configuration.md" >}}).
 The Grafana backend includes Sqlite3 which requires GCC to compile. So
 in order to compile Grafana on Windows you need to install GCC. We
 recommend [TDM-GCC](http://tdm-gcc.tdragon.net/download).
+
+## Logging in for the first time
+
+To run Grafana open your browser and go to port 3000 which is the default port. If you have changed the port you go to that port. There you will see the login page. User name is admin and password is admin. When you log in for the first time you will be asked to change your password. You can later go to user preferences and change your user name.
```
Comment (marefr) [inline docs/sources/installation/windows.md]: > To run Grafana open your browser and go to port 3000

To run Grafana open your browser and go to the port you configured [above](/installation/windows/#configure), e.g. `http://localhost:8080`.

Would be really nice if each installation guideline links to the getting started page. Maybe possible then to shorten the "Logging in for the first time" for each platform and have the details in getting started. What do you think?

**Review by marefr — COMMENTED** [review]
Much better than before. Have some comments and suggestions.

**Review by marefr — COMMENTED** [review]
Much better. I pushed some changes to make it a bit more clear what 3000 is and link to how you configure that - not sure if that's to much details?

Also did some changes to getting started page and removed the prerequisite of having a datasource setup and working.

See changes: https://github.com/grafana/grafana/pull/12919/commits/b218290f3ae28c2109f4a67039ae922319caaa9b

**Comment by marefr** [comment]
@Ijin08 have you had a chance to look at my last commit?

**Comment by Ijin08** [comment]
@marefr Your commit looks good. Much clearer.

**Pull request merged**